src/riak_cs_config.erl

```diff
@@ -402,6 +403,15 @@ read_before_last_manifest_write() ->
 region() ->
     get_env(riak_cs, region, ?DEFAULT_REGION).
 
+%% @doc This switch changes mapreduce input from listkeys to
+%% fold_objects after riak_kv's mapreduce input optimization
+%% merged and enabled. With fold_objects it will have 1/2 IO
+%% when scanning the whole bucket.
+-spec use_2i_for_storage_calc() -> boolean().
+use_2i_for_storage_calc() ->
+    riak_cs_list_objects_utils:fold_objects_for_list_keys()
+        andalso get_env(riak_cs, use_2i_for_storage_calc, false).
```

Comment:
Index query by `$bucket` is not 2i-ish, to my mind :sweat_smile: 
`fold_objects_for_storage_calc` is more sensible?